src/triton_gazebo/models/TLA/model.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<model>
+    <name>TLA</name>
```

Comment:
I am not sure if TLA is the best intuitive name for it, what it really is is the actual auv right? Maybe the name triton_auv or auv would be more understandable and descriptive of the model

Reply:
Looking at it now I agree with you, I think that Triton is the auv name so I think that would be a good fit.